marketplace/cloudinary-dialog/src/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+  </head>
+
+  <body>
+    <div id="root"></div>
+    <script src="https://media-library.cloudinary.com/global/all.js"></script>
```

Comment:
I wonder if we can use these dep as npm dependencies? It would eliminate two network requests.

Reply:
As far as I know, all.js is not part of any npm package that is provided by Cloudinary. That is the main reason why I added in the .html file.

Reply:
I've looked into this. It looks like the Cloudinary Media Library isn't available in one of the NPM packages provided by Cloudinary. Currently, it looks like that this is the only way to load it into the system. 

A way to eliminate the extra network call is to download the script and load it manually into the project. Personally, I'm not a huge fan of this, because I want to receive updates of all.js. What do you think?

Reply:
Let's leave it like this.